### PR TITLE
Final minor bug fixes.

### PR DESCRIPTION
**Pull request by Paul-Ferrell**

### CODE REVIEW

**Comment by Paul-Ferrell** [comment]
Fixes #103

File: test/utils/lock_test.py [hunk 1]
```diff
@@ -1,12 +1,40 @@
...
+"""
+Multi-host lockfile test.
+
+Usage: python3 lock_test.py <lockfile_dir>
+
+Run this simultaniously on multiple hosts that share an NFS filesystem 
+to test cross-system NFS locking. The tests do a synchronized start at the top 
+of the minute according to the system clock, so make sure the system clocks are close 
+(within .5 seconds will do). Any locking errors will be
+printed.
```
Comment (nicholas-sly) [inline test/utils/lock_test.py:10]: This could be on the previous line.

File: test/utils/lock_test.py [hunk 2]
```diff
@@ -1,12 +1,40 @@
...
+
+The shared lockfile is placed in this directory
```
Comment (nicholas-sly) [inline test/utils/lock_test.py:12]: No period at the end of this sentence.

Comment (Paul-Ferrell) [reply test/utils/lock_test.py:12]: This doesn't need to be super kosher.

**Pull request merged**